ArduCopter/mode.h

```diff
@@ -315,7 +315,7 @@ class ModeAcro : public Mode {
     const char *name() const override { return "ACRO"; }
     const char *name4() const override { return "ACRO"; }
 
-    void get_pilot_desired_angle_rates(int16_t roll_in, int16_t pitch_in, int16_t yaw_in, float &roll_out, float &pitch_out, float &yaw_out);
+    void get_pilot_desired_angle_rates(float roll_in, float pitch_in, float yaw_in, float &roll_out, float &pitch_out, float &yaw_out);
```

Comment:
Because we've touched the method let's put a comment above it to correct for past wrongs.  That comment should mention the range of the arguments because the argument names themselves don't make it at all clear.